vector-im/element-android
# Don't dismiss bottom sheet if verification is required

## Type of change


## Content
When verification is required by the server, self verification bottom sheet should not be dismissable.

## Motivation and context

[Closes #7735](https://github.com/vector-im/element-android/issues/7735)

## Screenshots / GIFs





## Tests



- Set `isVerificationRequired` to true in `SelfVerificationViewModel`.
- Try to do self verification, bottom-sheet should not be dismissable.

## Tested devices

- OS version(s): API 11 and 12

## Checklist
